meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-atf/imx-atf_%.bbappend

```diff
@@ -5,6 +5,7 @@ RPROVIDES_${PN} += "virtual/trusted-firmware-a"
 
 SRC_URI_append_toradex = " \
     file://0001-Revert-Add-NXP-s-SoCs-partition-reboot-support.patch \
+    file://0002-plat-imx8m-add-SiP-call-for-SRC-PERSIST_SECONDARY_BO.patch \
```

Comment:
Shouldn't be part of append_toradex, but SRC_URI_append instead.

Reply:
yes, didn't catch that as I'm building everything from my workspace. Will fix, thanks!